app/src/docs/v1.api-spec.yaml

```diff
@@ -4,9 +4,7 @@ info:
   version: 1.0.0
   title: Common Forms Toolkit API
   description: >-
-    An API for a Forms application that supports Form management and
-    configuration, Submission functions such as Status changes, Notes as well as
-    Admin Team management, Notifications and more.
+    An API for a Forms application. This API supports a complete set of Form Management functions, ranging from the validation and capture of form data to Administrative feature such as form submission Status changes, assigning users or adding Notes. The toolkit also offers Team access management, Notifications, Document generation and Reporting.
```

Comment:
As usual, make sure to use one of the auto-formatters to make sure it's word wrapping appropriately.